### ISSUE TITLE
Insertrow modal placeholders have commas when dynamically generating the table.

### ISSUE THREAD

**Issue by Arvin6** [AllenFang/react-bootstrap-table#1145]
This is probably because the <TableHeaderColumn/> gets its child through a map operation.

`this.props.keys.map( (key) =>  <TableHeaderColumn key={key} width={key==="sno"?"50px":"150px"} dataField={key}> {key.toUpperCase()} </TableHeaderColumn> )`

![screenshot from 2017-03-14 00-45-15](https://cloud.githubusercontent.com/assets/24292251/23871685/7519823e-0851-11e7-9d56-083842eb0357.png)

**Comment by AllenFang** [comment]
ok, I'll check this out, but I'm pretty busy so maybe need some time.

**Comment by Arvin6** [comment]
Also, I found another bug on mobile. I'll attach the screenshot. The new button overlaps on the insert modal.
![screenshot_20170321-164609](https://cloud.githubusercontent.com/assets/24292251/24145266/61c7dbbc-0e56-11e7-9d65-fcf210a220b6.png)

**Comment by rfinck** [comment]
@Arvin6 You can use the `headerText` prop to specify your placeholders so that they won't have commas.

```
this.props.keys.map( (key) => 
    <TableHeaderColumn
        key={key}
        width={key==="sno"?"50px":"150px"}
        dataField={key}
        headerText={key.toUpperCase()}>
        {key.toUpperCase()}
    </TableHeaderColumn> 
)
```

**Comment by Arvin6** [comment]
@rfinck thank you so much. It worked. :)

**Comment by AllenFang** [comment]
Fixed on `v3.1.6`.
